src/content/docs/start/prerequisites.mdx

```diff
@@ -265,7 +297,13 @@ If you'd like to target your app for Android or iOS then there are a few additio
 
 ### Android
 
-1. Download and install [Android Studio from the Android Developers website](https://developer.android.com/studio)
+:::caution[Nix/NixOS]
+Android Studio only supports Linux Nix/NixOS.
+If you are using Nix on darwin you could try the [homebrew route through nix-darwin](https://daiderd.com/nix-darwin/manual/index.html#opt-homebrew.brews) to install Android Studio.
+:::
+
+1. Download and install [Android Studio from the Android Developers website](https://developer.android.com/studio) or if you are using Nix/NixOS by adding the package to your system `android-studio`.
+
```

Comment:
This *really* doesn't need Nix/NixOS specific instructions. If you're using either, I am sure you know how to install a package. We don't need to explain that here

Reply:
Your right, Resolved.